### ISSUE TITLE
[question] What text preprocessing have you used for ASR models stt-eu, stt-de, stt-es, stt-fr. Is it only nemo text norm?

### ISSUE THREAD

**Issue by vbaghdasaryan** [NVIDIA/NeMo#10122]
I am interested in these models
stt_multilingual_fastconformer_hybrid_large_pc_blend_eu, 
 stt_de_fastconformer_hybrid_large_pc, 
 stt_es_fastconformer_hybrid_large_pc,  
 stt_fr_fastconformer_hybrid_large_pc.

Its intuitive to think that you used nemo text processing package for text norm. But I tried to finetune multilingual  and german models and got worse results. 
I downloaded the same datasets, used your NeMo codebase, your published checkpoint and config
Tried to add some more new public data. Kept the learning rate lower than your minimum in the config.
After training a bit, I test on fleurs test dataset and get worse results. 

### My concern is that maybe you didn't use nemo text processing or used something else on top of it or before.
That would very much help my project if you clarified that part. 
I think that resuming training on same data/ same + some more data couldn't damage that much with small learning rate.
German stt model from 8% jumped to 11% after an epoch.

Thank you in advance

**Comment by erastorgueva-nv** [comment]
Hi, thank you for your question.

You can see the code used to clean the Spanish data here: https://github.com/NVIDIA/NeMo-speech-data-processor/tree/main/dataset_configs/spanish_pc.

The rules used to clean the German and French data were very similar.

Actually, IIRC, we did not use NeMo Text Processing at that time. You can see there some custom rules for handling Roman Numerals, handling common abbreviations like "Dr".

For Voxpopuli, there was normalized text without punctuation and capialization, we made a processor to add the normalization to the version of the text with punctuation and capitalization: https://github.com/NVIDIA/NeMo-speech-data-processor/blob/main/sdp/processors/datasets/voxpopuli/normalize_from_non_pc_text.py#L118.

In general, a lot of care was taken to clean each specific dataset. We browsed the data and looked for common issues using the NeMo [Speech Data Explorer](https://github.com/NVIDIA/NeMo/tree/main/tools/speech_data_explorer) tool.
